src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,85 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
 import { Loader } from '../Loader';
+import { useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { actions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const [user, setUser] = useState<User>({
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+  });
+
+  useEffect(() => {
+    if (currentTodo?.id) {
+      getUser(currentTodo.id).then((result: User) => setUser(result))
+        .catch(() => setUser({
+          id: currentTodo.userId,
+          name: '',
+          email: '',
+          phone: '',
+        }));
+    }
+  }, [currentTodo]);
+
+  const dispatch = useDispatch();
+
+  const removeCurrentTodo = () => dispatch(actions.removeTodo());
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      <Loader />
+      {user?.id
+        ? (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${currentTodo?.id}`}
+              </div>
 
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Remove

Reply:
I cannot do it here because label for button will be required by linter. And I haven't added this disabling statement